client/src/app/site/polls/components/base-poll-detail.component.ts

```diff
@@ -98,9 +107,23 @@ export abstract class BasePollDetailComponent<V extends ViewBasePoll> extends Ba
         protected groupRepo: GroupRepositoryService,
         protected promptService: PromptService,
         protected pollDialog: BasePollDialogService<V>,
-        protected pollService: PollService
+        protected pollService: PollService,
+        protected votesRepo: BaseRepository<ViewMotionVote | ViewAssignmentVote, MotionVote | AssignmentVote, object>
```

Comment:
This class should not know ViewMotionVote, nor ViewAssignmentVote, nor MotionVote, nor AssignmentVote
Use generics / templates